core/src/main/scala/org/apache/spark/sql/extensions/rules.scala

```diff
@@ -26,7 +26,8 @@ class TiAuthRuleFactory(getOrCreateTiContext: SparkSession => TiContext)
     extends (SparkSession => Rule[LogicalPlan]) {
   private val logger = LoggerFactory.getLogger(getClass.getName)
   override def apply(sparkSession: SparkSession): Rule[LogicalPlan] = {
-    if (TiExtensions.authEnable(sparkSession)) {
+    // auth only work for catalog plugin mode
+    if (TiExtensions.catalogPluginMode(sparkSession) && TiExtensions.authEnable(sparkSession)) {
```

Comment:
From my point of view, we should rename catalogPluginMode to validateCatalog, Since we don't support without catalogMode.

Reply:
good idea